src/styles/blocks/card.scss

```diff
@@ -0,0 +1,104 @@
+.card {
+  font-family: $font-family-main;
+  box-sizing: border-box;
+  width: 200px;
+  padding: 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  transition: transform 300ms;
+
+  &__product-image {
+    display: flex;
+    width: 160px;
+    height: 134px;
+    padding: 16px 3px 0;
+  }
+
+  &__product-name {
+    margin: 0;
+    padding-top: 40px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $text-color-primary;
+  }
+
+  &__product-code {
+    margin: 0;
+    padding-top: 4px;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $text-color-secondary;
+  }
+
+  &__product-rating {
+    padding-top: 16px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+
+    &-reviews {
+      margin: 0;
+      display: flex;
+      font-size: 10px;
+      line-height: 14px;
+      font-weight: 400;
+      color: $text-color-primary;
+      align-self: flex-end;
+    }
+  }
+
+  &__product-price {
+    padding-top: 24px;
+    padding-bottom: 16px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    line-height: 18px;
+
+    &-text {
+      margin: 0;
+      font-size: 12px;
+      font-weight: 400;
+      color: $text-color-secondary;
+    }
+
+    &-numbers {
+      margin: 0;
+      font-size: 16px;
+      font-weight: 700;
+      color: $text-color-primary;
+    }
+  }
+
+  &__product-buy-button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    height: 40px;
+    border-radius: 5px;
+    background-color: $primary-color;
+    font-size: 14px;
+    line-height: 16px;
+    font-weight: 700;
+    text-decoration: none;
+    color: $secondary-color;
+
+    &:hover {
+      background-color: $secondary-color;
+      color: $primary-color;
+      border-style: solid;
+      border-width: 1px;
+      border-color: $primary-color;
```

Comment:
The hover effect for the &__product-buy-button should maintain consistency with the design. Ensure that the hover styles align with the design specifications provided in the task description.